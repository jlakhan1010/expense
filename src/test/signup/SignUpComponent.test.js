import React from 'react';
import { shallow } from 'enzyme';
import * as enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import SignUp from '../../signup/SignUpComponant';

(enzyme).configure({ adapter: new Adapter() })
const shallowWithStore = (component, store) => {
  const context = {
    store,
  };
  return shallow(component, { context });
};

export default shallowWithStore;

describe("SignUp component", () => {
    it("Should match snapshot", () => {
        const wrapper = shallow(<SignUp />);
        expect(wrapper).toMatchSnapshot();
    });
});
