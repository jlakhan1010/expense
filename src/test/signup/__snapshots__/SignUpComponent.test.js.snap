exports[`SignUp component Should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <[object Object]
    destroyOnUnmount={true}
    enableReinitialize={false}
    forceUnregisterOnUnmount={false}
    form="signupForm"
    getFormState={[Function getFormState]}
    keepDirtyOnReinitialize={false}
    persistentSubmitErrors={false}
    pure={true}
    shouldAsyncValidate={[Function defaultShouldAsyncValidate]}
    shouldError={[Function defaultShouldError]}
    shouldValidate={[Function defaultShouldValidate]}
    shouldWarn={[Function defaultShouldWarn]}
    touchOnBlur={true}
    touchOnChange={false}
    updateUnregisteredFields={false} />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function batchedUpdates],
    "getNode": [Function getNode],
    "render": [Function render],
    "simulateEvent": [Function simulateEvent],
    "unmount": [Function unmount]
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "destroyOnUnmount": true,
      "enableReinitialize": false,
      "forceUnregisterOnUnmount": false,
      "form": "signupForm",
      "forwardedRef": null,
      "getFormState": [Function getFormState],
      "keepDirtyOnReinitialize": false,
      "persistentSubmitErrors": false,
      "pure": true,
      "shouldAsyncValidate": [Function defaultShouldAsyncValidate],
      "shouldError": [Function defaultShouldError],
      "shouldValidate": [Function defaultShouldValidate],
      "shouldWarn": [Function defaultShouldWarn],
      "touchOnBlur": true,
      "touchOnChange": false,
      "updateUnregisteredFields": false
    },
    "ref": null,
    "rendered": null,
    "type": [Function Hoc]
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "destroyOnUnmount": true,
        "enableReinitialize": false,
        "forceUnregisterOnUnmount": false,
        "form": "signupForm",
        "forwardedRef": null,
        "getFormState": [Function getFormState],
        "keepDirtyOnReinitialize": false,
        "persistentSubmitErrors": false,
        "pure": true,
        "shouldAsyncValidate": [Function defaultShouldAsyncValidate],
        "shouldError": [Function defaultShouldError],
        "shouldValidate": [Function defaultShouldValidate],
        "shouldWarn": [Function defaultShouldWarn],
        "touchOnBlur": true,
        "touchOnChange": false,
        "updateUnregisteredFields": false
      },
      "ref": null,
      "rendered": null,
      "type": [Function Hoc]
    }
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true
      }
    },
    "disableLifecycleMethods": true
  }
}
`;
